README.md

```diff
@@ -33,6 +33,13 @@ $ cp .env.template .env  # (first time only)
 ```
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
+There are four varaibles for trello access
+APIKEY
+APIToken
+TrelloBID
+URL = "https://api.trello.com/"
```

Comment:
Thanks for documenting these - it would be good to also add them to your `.env.template` as well so that there's an easy prompt for new users there too

Separately, it's worth trying to stick to no-spaces and no-quotes as the format for your variables, e.g.
```
URL=https://api.trello.com/
```

Python is forgiving but you may find that it bites you later as Docker will interpret that as a string that actually includes quotes